tests/unit/components/sl-drop-option-test.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-drop-option', 'Unit - component: sl-drop-option' );
+
+test( 'Has expected initial class name', function( assert ) {
+    var $component = this.render();
+
+    assert.ok(
+        $component.hasClass( 'sl-drop-option' ),
+        'Rendered component has class "sl-drop-option"'
+    );
+});
+
+test( 'Has expected aria-role property', function( assert ) {
+    var $component = this.render();
+
+    assert.strictEqual(
+        $component.attr( 'role' ),
+        'menuitem',
+        'ARIA role is properly set to "menuitem"'
+    );
+});
+
+test( 'Option type class value depends on `label` value', function( assert ) {
+    var component  = this.subject(),
+        $component = this.render();
+
+    assert.ok( $component.hasClass( 'divider' ), 'Initially has "divider"' );
+
+    Ember.run( function() {
+        component.set( 'label', 'Test' );
+    });
+
+    assert.ok(
+        $component.hasClass( 'presentation' ),
+        'Has class "presentation" with valid "label" value'
+    );
+});
```

Comment:
Do we want two additional tests that test the counter assertions of the current ones?  To make sure that the only class applied in the situation is the desired one.  As currently tested it would be possible for optionType to be modified to apply multiple classes and the tests would still pass, as long as the one of them matched the assertion.  While technically this might be okay because DOM elements can have multiple classes, specifically we want to ensure that this element cannot have both the "divider" and "presentation" class applied at the same time.

This inquiry is mostly valid, partially a question of philosophy of our approach.